docs/semgrep-assistant/getting-started.md

```diff
@@ -139,6 +160,29 @@ To add a memory:
 
 While Assistant Memories is in **public beta**, memories are scoped to remediation guidance on a per-project and per-rule basis. A saved memory only affects future guidance for findings triggered by the same rule in the same project.
 
+### Add memory during triage
+
+If you identify findings that are safe to ignore and write triage notes indicating why this is so, Assistant can store this information as a memory and use it when assessing whether a similar finding should be shown to developers in the future. Assistant also takes that memory, reanalyzes similar findings in your backlog, and suggests issues that may be safe to close.
+
+To add a memory during triage:
+
+1. Identify the specific finding you want to modify, and open up its finding details page.
+2. Change the status of the finding to **Ignored**, and optionally, select an **Ignore reason** and provide **Comments** on why you're changing the finding's status as **Ignored**.
+3. Click **Ignore & add memory**. 
+4. In the **Create memory** pop-up window:
```

Comment:
```suggestion
4. In the **Create memory** dialog box:
```
- either dialog or dialog box is ok
- pop-up window is definitely a Microsoft term 😅 
![Snag_15e1be](https://github.com/user-attachments/assets/53c5398f-11d6-46a2-a2f7-a2b8b50a4e5b)